taskvine/src/manager/vine_worker_info.h

```diff
@@ -42,9 +43,10 @@ struct vine_worker_info {
 	char *hashkey;
 
 	/* Address and port where this worker will accept transfers from peers. */
-	char transfer_addr[LINK_ADDRESS_MAX];
+	char transfer_addr[DOMAIN_NAME_MAX];
```

Comment:
If this is going to store a domain name, it should be called `transfer_host` now.